sources/logdog/src/commands.rs

```diff
@@ -0,0 +1,154 @@
+//! Provides the list of commands that `logdog` will run and provides the handler function to run
+//! them.
+//!
+//! # Static Commands
+//!
+//! At build time, the `build.rs` file checks which variant is being built and creates a symlink
+//! file which points to the commands for the current variant. This file is named `variant`. We load
+//! the `variant` and `common` files into static strings at compile time, and these provide the list
+//! of commands that `logdog` will run.
+
+use crate::error::{self, Result};
+use reqwest::blocking::{Client, Response};
+use snafu::{OptionExt, ResultExt};
+use std::fs::File;
+use std::path::Path;
+use std::process::Stdio;
+use url::Url;
+
+/// The `logdog` commands that all variants have in common.
+const COMMON_COMMANDS: &str = std::include_str!("../conf/logdog.common.conf");
+/// The `logdog` commands that are specific to the current variant.
+const VARIANT_COMMANDS: &str = std::include_str!("../conf/current/logdog.conf");
+
+/// Returns the list of commands to run by combining `COMMON_COMMANDS` and `VARIANT_COMMANDS`.
+/// These are read at compile-time from files named `common` and `variant` respectively.
+pub(crate) fn commands() -> Vec<&'static str> {
+    let mut commands: Vec<&str> = COMMON_COMMANDS
+        .lines()
+        .filter(|&command| !command.is_empty())
+        .collect();
+    let mut variant_commands: Vec<&str> = VARIANT_COMMANDS
+        .lines()
+        .filter(|&command| !command.is_empty())
+        .collect();
+    commands.append(&mut variant_commands);
+    commands
```

Comment:
```suggestion
    COMMON_COMMANDS
        .lines()
        .chain(VARIANT_COMMANDS
        .lines())
        .filter(|&command| !command.is_empty() && !command.trim_start().starts_with('#'))
        .collect()
```
Shouldn't need to construct/expand two vecs and filter twice.

(Also fixes Ben's request)

Reply:
[aa7d592](https://github.com/bottlerocket-os/bottlerocket/compare/60e5d3c8e65c58170e8f72d944a80be11ea098e0..aa7d59202dad5161a9dc8aff3995f1fbe0d4c706)